doc/source/user/troubleshooting-importerror.rst

```diff
@@ -133,10 +133,10 @@ your system. If you can open a correct python shell, you can also run the
 following in python::
 
     import os
-    PYTHONPATH = os.environ['PYTHONPATH'].split(os.pathsep)
-    print("The PYTHONPATH is:", PYTHONPATH)
-    PATH = os.environ['PATH'].split(os.pathsep)
-    print("The PATH is:", PATH)
+    paths = os.environ.get('PYTHONPATH')
+    print("PYTHONPATH:", paths.split(os.pathsep) if paths else "empty")
+    paths = os.environ.get('PATH')
+    print("PATH:", paths.split(os.pathsep) if paths else "empty")
```

Comment:
Note that on Windows this is quite unfriendly, as all the parts contain backslashes, and printing a `list` shows the repr complete with escape sequences

Reply:
Eeek. So `", ".join` or just print directly?

Reply:
nvm. lets just print them directly and if someone complains its confusing we can try to be smart about it...

Reply:
`"\n".join` might be reasonable. Of course, then it becomes hard to spot empty strings in the path (which mean cwd)